drift/app.py

```diff
@@ -24,6 +24,7 @@ def create_app():
     connexion_app = connexion.App(__name__, specification_dir="openapi/", arguments=openapi_args)
     connexion_app.add_api("api.spec.yaml", validate_responses=True, strict_validation=True)
     connexion_app.add_api("mgmt_api.spec.yaml")
+    connexion_app.add_api("admin_api.spec.yaml", validate_responses=True, strict_validation=True)
```

Comment:
In this file, you will also need to import similar to here in baselines:
https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/app.py#L15

And you will need to register the blueprint as well I think:
https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/app.py#L56

See this PR for further comparisons of how this was done for the internal endpoint:
https://github.com/RedHatInsights/system-baseline-backend/pull/179/files

Reply:
fixed.